Core/src/org/sleuthkit/autopsy/centralrepository/persona/AddAccountDialog.java

```diff
@@ -0,0 +1,337 @@
+                        .addContainerGap(202, Short.MAX_VALUE)
+                        .addComponent(okBtn)
+                        .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
+                        .addComponent(cancelBtn, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)))
+                .addContainerGap())
+        );
+
+        layout.linkSize(javax.swing.SwingConstants.HORIZONTAL, new java.awt.Component[] {cancelBtn, okBtn});
+
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addComponent(settingsPanel, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE)
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED, javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE)
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+                    .addComponent(okBtn, javax.swing.GroupLayout.Alignment.TRAILING)
+                    .addComponent(cancelBtn, javax.swing.GroupLayout.Alignment.TRAILING, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
+                .addContainerGap(javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
+        );
+
+        pack();
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void display() {
+        this.setLocationRelativeTo(WindowManager.getDefault().getMainWindow());
+        setVisible(true);
+    }
+
+    @Messages({
+        "AddAccountDialog_dup_Title=Account add failure",
+        "AddAccountDialog_dup_msg=This account is already added to the persona",
+        "AddAccountDialog_empty_Title=Empty identifier",
+        "AddAccountDialog_empty_msg=The identifier field cannot be empty",
+        "AddAccountDialog_search_failure_Title=Account add failure",
+        "AddAccountDialog_search_failure_msg=Central Repository account search failed",
+        "AddAccountDialog_search_empty_Title=Account not found",
+        "AddAccountDialog_search_empty_msg=Account not found for given identifier and type",})
+    private void okBtnActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_okBtnActionPerformed
+        if (identifierTextField.getText().isEmpty()) {
+            JOptionPane.showMessageDialog(this,
+                    Bundle.AddAccountDialog_empty_msg(),
+                    Bundle.AddAccountDialog_empty_Title(),
+                    JOptionPane.ERROR_MESSAGE);
+            return;
+        }
+        Collection<PersonaAccount> candidates;
+        try {
+            candidates = 
+                    PersonaAccount.getPersonaAccountsForIdentifierLike(identifierTextField.getText());
```

Comment:
We added these  apis to CentralRepoAccount:
 - getAccountsWithIdentifierLike()
 - getAccountsWithIdentifier()

I think when looking for an account to create a persona, those are the ones we should use here.  

Also it isn't obvious if the user is supposed to type in a substring or a full account identifier.  We are using the api getAccountsWithIdentifierLike().  What if that returns multiple hits  - does the user get to pick one?